pkg/collector/python/loader.go

```diff
@@ -271,13 +271,13 @@ func reportPy3Warnings(checkName string, checkFilePath string) {
 			metricValue = 1.0
 		} else {
 			status = a7TagNotReady
-			log.Warnf("The Python 3 linter returned warnings for check '%s'. Set the log level to \"debug\" and restart the Agent to have the linter warnings logged.", checkName)
+			log.Warnf("The Python 3 linter returned warnings for check '%s'. For more details, check the output of the "status" command or the status page of the Agent GUI).", checkName)
```

Comment:
CI fails because of this I think:

```suggestion
			log.Warnf("The Python 3 linter returned warnings for check '%s'. For more details, check the output of the 'status' command or the status page of the Agent GUI).", checkName)
```